From 9c350c037ca3489dbeece6ecc2d7e2e5dbb177e9 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Sat, 11 May 2019 15:21:46 +0800
Subject: [PATCH] These tests don't work in cross compiling, just disable them
 for now, we don't build subversion on OS-X at this time.

RP 1014/7/16

Upstream-Status: Pending [needs a rewrite to support a cache value]

Rebase to 1.12.0

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 build/ac-macros/macosx.m4 | 31 +------------------------------
 1 file changed, 1 insertion(+), 30 deletions(-)

diff --git a/build/ac-macros/macosx.m4 b/build/ac-macros/macosx.m4
index 92fa58e..a568e1c 100644
--- a/build/ac-macros/macosx.m4
+++ b/build/ac-macros/macosx.m4
@@ -24,21 +24,7 @@ dnl Check for _dyld_image_name and _dyld_image_header availability
 AC_DEFUN(SVN_LIB_MACHO_ITERATE,
 [
   AC_MSG_CHECKING([for Mach-O dynamic module iteration functions])
-  AC_RUN_IFELSE([AC_LANG_PROGRAM([[
-    #include <mach-o/dyld.h>
-    #include <mach-o/loader.h>
-  ]],[[
-    const struct mach_header *header = _dyld_get_image_header(0);
-    const char *name = _dyld_get_image_name(0);
-    if (name && header) return 0;
-    return 1;
-  ]])],[
-    AC_DEFINE([SVN_HAVE_MACHO_ITERATE], [1],
-              [Is Mach-O low-level _dyld API available?])
-    AC_MSG_RESULT([yes])
-  ],[
     AC_MSG_RESULT([no])
-  ])
 ])
 
 dnl SVN_LIB_MACOS_PLIST
@@ -46,23 +32,8 @@ dnl Assign variables for Mac OS property list support
 AC_DEFUN(SVN_LIB_MACOS_PLIST,
 [
   AC_MSG_CHECKING([for Mac OS property list utilities])
-
-  AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[
-    #include <AvailabilityMacros.h>
-    #if !defined(MAC_OS_X_VERSION_MAX_ALLOWED) \
-     || !defined(MAC_OS_X_VERSION_10_0) \
-     || (MAC_OS_X_VERSION_MAX_ALLOWED <= MAC_OS_X_VERSION_10_0)
-    #error ProperyList API unavailable.
-    #endif
-  ]],[[]])],[
-    SVN_MACOS_PLIST_LIBS="-framework CoreFoundation"
-    AC_SUBST(SVN_MACOS_PLIST_LIBS)
-    AC_DEFINE([SVN_HAVE_MACOS_PLIST], [1],
-              [Is Mac OS property list API available?])
-    AC_MSG_RESULT([yes])
-  ],[
+  AC_SUBST([SVN_MACOS_PLIST_LIBS], [""])
     AC_MSG_RESULT([no])
-  ])
 ])
 
 dnl SVN_LIB_MACOS_KEYCHAIN
-- 
2.7.4

